compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -413,6 +413,10 @@ private void defineConstructs(BLangPackage pkgNode, SymbolEnv pkgEnv) {
         if (!PackageID.ANNOTATIONS.equals(pkgNode.packageID)) {
             initPredeclaredModules(symTable.predeclaredModules, pkgNode.compUnits, pkgEnv);
         }
+        
+        if (REGEXP.equals(pkgNode.packageID)) {
```

Comment:
Did you double-check whether loading happens correctly even without this change?